### DISCUSSION TITLE
Sub flow would be very nice.

### DISCUSSION THREAD

**Discussion by nerkasnimo** [FlowiseAI/Flowise#1712] [Ideas]
Hello. This is very easy to use. Good job Henry. thank you.
I want to add some idea for this. 
if there is a "sub flow" functionality like [this](https://reactflow.dev/examples/layout/sub-flows) , it would be very good, wouldnt be.

**Comment by dcsan** [comment]
Is there anyway to save and link across separate flows right now? 

I think once you get beyond a simple prototype your flowchart quickly becomes the size of a building, so being able to split and link flows seems critical. they don't have to be cute little subflows in the UI but even linking by name somehow?

**Comment by chungyau97** [comment]
Hi @nerkasnimo & @dcsan,

Added your feedback into our backlog.
